allocate.py

```diff
@@ -1,13 +1,11 @@
 #!/usr/bin/python -u
 
-import os
 import sys
 from collections import Counter
-
```

Comment:
third party imports should be separately grouped. See https://www.python.org/dev/peps/pep-0008/#imports